.github/workflows/linux_builds.yml

```diff
@@ -138,6 +138,22 @@ jobs:
         run: |
           ./bin/godot.linuxbsd.tools.64s --test
 
+      # Download and test project to check leaks and invalid memory usage.
+      # CI has no audio device, so use the Dummy audio driver to avoid spurious error messages.
+      - name: Importing and running project
+        run: |
+          wget2 https://github.com/qarmin/RegressionTestProject/archive/master.zip
```

Comment:
Not a big deal i guess, but it's probably better to host the test project in a `github.com/godotengine` repository at some point.

Reply:
Godot should have an official test project with a clear vision of what it should look like, well thought-out logic, simplicity (allowing for easy understanding, modernization of the project and updating in case of breaking changes).

My project(RegressionTestProject) is created of several projects and serves me a bit as a testing ground (builds 1h long or with thread sanitizer), so its quality isn't best but maybe someone with a vision (which I don't have) will be able to create a project based on it, with better quality which would be able to become official and will replace current one.